hbase-server/src/main/resources/hbase-webapps/regionserver/processRS.jsp

```diff
@@ -68,8 +70,8 @@ pageContext.setAttribute("pageTitle", "Process info for PID: " + JSONMetricUtil.
         <td><%= new Date(runtimeBean.getStartTime()) %></td>
         <td><%= StringUtils.humanTimeDiff(runtimeBean.getUptime()) %></td>
         <td><%= JSONMetricUtil.getProcessPID() %></td>
-        <td><%= (long)JSONMetricUtil.getValueFromMBean(rsMetrics, "pauseWarnThresholdExceeded")
-          + (long)JSONMetricUtil.getValueFromMBean(rsMetrics, "pauseInfoThresholdExceeded") %></td>
+        <td><%= pauseWarnThresholdExceeded != null && pauseInfoThresholdExceeded != null ?
+                (long)pauseWarnThresholdExceeded + (long)pauseInfoThresholdExceeded : -1 %></td>
```

Comment:
Is it better to display 0 if it's null?  I'm not sure, maybe we can hear other people's opinions?

`<th>JvmPauseMonitor Count </th>` , Can you also remove this space in line65?

Reply:
Happy to replace with whatever is preferred. How about the string "unknown"?

Reply:
Changed the value to 0 when the metrics are null. Add fixed the spacing on line 65.

Reply:
I have no more questions. let's wait for others to review.